charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -20,3 +20,1580 @@ spec:
+                      properties:
+                        pullPolicy:
+                          type: string
+                        pullSecrets:
+                          type: array
+                        repository:
+                          type: string
+                        tag:
+                          type: string
+                    labels:
+                      type: object
+                    metastore:
+                      type: object
+                      properties:
+                        affinity:
+                          type: object
+                        config:
+                          type: object
+                          properties:
+                            auth:
+                              type: object
+                              properties:
+                                enabled:
+                                  type: boolean
+                            jvm:
+                              type: object
+                              properties:
+                                initialRAMPercentage:
+                                  type: string
+                                maxRAMPercentage:
+                                  type: string
+                                minRAMPercentage:
+                                  type: string
+                            logLevel:
+                              type: string
+                            tls:
+                              type: object
+                              properties:
+                                caCertificate:
+                                  type: string
+                                certificate:
+                                  type: string
+                                createSecret:
+                                  type: boolean
+                                enabled:
+                                  type: boolean
+                                key:
+                                  type: string
+                                secretName:
+                                  type: string
+                        livenessProbe:
+                          type: object
+                          properties:
+                            failureThreshold:
+                              type: integer
+                            initialDelaySeconds:
+                              type: integer
+                            periodSeconds:
+                              type: integer
+                            successThreshold:
+                              type: integer
+                            tcpSocket:
+                              type: object
+                              properties:
+                                port:
+                                  type: integer
+                            timeoutSeconds:
+                              type: integer
+                        nodeSelector:
+                          type: object
+                        readinessProbe:
```

Comment:
Could potentially steal the structure for this from https://github.com/coreos/prometheus-operator/blob/master/example/prometheus-operator-crd/prometheus.crd.yaml#L1310 and do the same with liveness.